app/scripts/views/choose_what_to_sync.js

```diff
@@ -21,6 +20,8 @@ define(function (require, exports, module) {
     initialize: function () {
       // Account data is passed in from sign up flow.
       this._account = this.user.initAccount(this.model.get('account'));
+
+      this.next = this.onSignUpSuccess.bind(this);
```

Comment:
nit: not sure about the naming `next`, any way to rename this to what `nextThing` this is ?